spring-cloud-generator/pom.xml

```diff
@@ -18,7 +18,7 @@
         <gapic-generator-java-bom.version>2.20.0</gapic-generator-java-bom.version>
         <!-- [gapic-test]: protobuf.version for compiling protos used in unit testing
              this should be consistent with version in gapic-generator-java-bom -->
-        <protobuf.version>3.21.12</protobuf.version>
+        <protobuf.version>3.23.1</protobuf.version>
```

Comment:
Isn't this the broken version?

Reply:
Ah, I missed the conversation around the re-release when putting out this PR, thanks for flagging. Will close this for now and wait for the new gapic-generator-java release.